tests/steps/common.py

```diff
@@ -134,3 +134,19 @@ def install_ch_tools_config_with_version(context, node):
     container.exec_run(
         ["bash", "-c", f"echo '{config_data}' > /etc/clickhouse-tools/config.yaml"]
     )
+
+
+@given("installed clickhouse-tools config with user on {nodes}")
+def install_ch_tools_config_with_user(context, nodes):
+    nodes_list = nodes.split(",")
+    user = "_admin"
+    password = ""
+
+    for node in nodes_list:
+        container = docker.get_container(context, node)
+        config_data = yaml.dump({"clickhouse": {"user": user, "password": password}})
+
+        container.exec_run(["bash", "-c", "mkdir /etc/clickhouse-tools"])
+        container.exec_run(
+            ["bash", "-c", f"echo '{config_data}' > /etc/clickhouse-tools/config.yaml"]
+        )
```

Comment:
I think it is better to do some generic method that will set something in config. 
It can look like that
```
Scenario: Check clickhouse orphaned objects with state-zk-path option
    Given clickhouse-tools configuration
    """
        clickhouse:
            user: "_admin"
            password: ""
    """
    When we execute command on clickhouse01
    .....
```
`step_configuration(context):` is already working this way.

Reply:
done